src/components/Profile/Profile/FeatureContainer/DoctorsFeature/WorkTimeManagement/WorkTimeManagement.tsx

```diff
@@ -44,23 +77,24 @@ const WorkTimeManagement: React.FC = () => {
             onChange={(newValue: Date | null) => {
               setDate(newValue);
             }}
-            inputFormat="dd.MM.yyyy hh:mm a"
-            minutesStep={5}
+            inputFormat="dd.MM.yyyy HH:mm a"
             shouldDisableDate={disableWeekends}
-            minTime={minTime}
-            maxTime={maxTime}
+            minTime={new Date('01/01/2021 08:00 AM')}
+            maxTime={new Date('01/01/2021 06:00 PM')}
           />
         </LocalizationProvider>
-        {date ? (
-          <Button onClick={() => addDate()} round size="small" variant="outlined">
-            add
-          </Button>
-        ) : (
-          <Button disabled onClick={() => addDate()} round size="small" variant="outlined">
-            add
-          </Button>
-        )}
+        {addDateError && <span>{addDateError}</span>}
       </DateTimePickerWrapper>
+
+      {date ? (
+        <Button onClick={() => addNewDate()} round size="small" variant="outlined">
+          add
+        </Button>
+      ) : (
+        <Button disabled round size="small" variant="outlined">
+          add
+        </Button>
```

Comment:
buttons are identical and differs only with one prop, we should render only one component

Reply:
fixed